### PR TITLE
gitea support

### LINKED ISSUE #133: add support for fetchFromGitea
We should also support gitea that for example https://github.com/NixOS/nixpkgs/pull/208964/files can be updated.

### PR DESCRIPTION
**Pull request by figsoda**
closes #133

### CODE REVIEW

File: nix_update/update.py [hunk 1]
```diff
@@ -174,6 +174,9 @@ def update_version(
         if package.parsed_url.netloc == "github.com":
             _, owner, repo, *_ = package.parsed_url.path.split("/")
             package.diff_url = f"https://github.com/{owner}/{repo}/compare/{package.rev}...{new_version.rev or new_version.number}"
+        elif package.parsed_url.netloc in ["codeberg.org", "gitea.com", "notabug.org"]:
```
Comment (SuperSandro2000) [inline nix_update/update.py:177]: Maybe we can detect if a remote is a gitea or gitlab automatically but probably something for another PR.

Comment (figsoda) [reply nix_update/update.py:177]: would be nice to do this detection only once, but yeah I agree this diserves separate PR

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
bors cancel

**Comment by Mic92** [comment]
bors try

**Comment by Mic92** [comment]
bors merge

**Pull request merged**